image_plane.go

```diff
@@ -144,3 +142,17 @@ func deblocking(outputBlocks [][]*ImagePlane, blocksW, blocksH int) []*ImagePlan
 	}
 	return outputPlanes
 }
+
+func (p ImagePlane) NewPixels() *Pixels {
```

Comment:
__[golint]__
* [golint] exported method ImagePlane.NewPixels should have comment or be unexported